### PR TITLE
Pageobjects

### PR DESCRIPTION
**Pull request by AndraP97**
Testing Amazon web site:
1. AddRemoveItemTests
2. DeliverToTests
3. FilteringSearchingTests
4. SearachFieldTests

### CODE REVIEW

File: src/main/java/org/example/pageobject/pages/CartPage.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    public CartPage deleteItem(){
+        delete.click();
+        return new CartPage(webDriver);
+    }
+    public String getCartMessage(){
+        return cartMessage.getText();
+    }
+
+    public CartPage waitForPageLoad() {
```
Comment (andreidbr) [inline src/main/java/org/example/pageobject/pages/CartPage.java]: This technically isn't waiting for page load, it's waiting for the appearance of the Shopping Cart status text

File: src/main/java/org/example/pageobject/pages/HomePage.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        return locationLabel.getText();
+    }
+
+    public void clickCountriesDropDown() {
+        var dropDown = new WebDriverWait(webDriver, Duration.ofSeconds(5))
+                .until(ExpectedConditions.elementToBeClickable(countriesDropDown));
+        dropDown.click();
+    }
+
+    public HomePage waitForPageLoad() {
```
Comment (andreidbr) [inline src/main/java/org/example/pageobject/pages/HomePage.java]: Again, not exactly waiting for page load :)

File: src/main/java/org/example/pageobject/pages/ResultsPage.java [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    WebElement textErorr;
+
+    @FindBy(xpath = "//span[@class=\"a-color-state a-text-bold\"]")
+    WebElement textEqualsWithKey;
+
+    public ResultsPage(WebDriver webDriver) {
+        super(webDriver);
+    }
+
+    public ProductPage selectAProduct() {
```
Comment (andreidbr) [inline src/main/java/org/example/pageobject/pages/ResultsPage.java]: I would put selectARandomProduct to specify that it is random selection

**Review by andreidbr — APPROVED** [review]

File: src/main/java/org/example/pageobject/pages/ResultsPage.java (hunk 3 above)
Comment (postolachedan) [reply src/main/java/org/example/pageobject/pages/ResultsPage.java]: Modified.

File: src/main/java/org/example/pageobject/pages/HomePage.java (hunk 2 above)
Comment (postolachedan) [reply src/main/java/org/example/pageobject/pages/HomePage.java]: changed method name to waitForLocationUpdate()

File: src/main/java/org/example/pageobject/pages/CartPage.java (hunk 1 above)
Comment (andrei-stoean) [reply src/main/java/org/example/pageobject/pages/CartPage.java]: Changed the name of the method to "waitForCartStatusText"

**Pull request merged**